playwright/e2e/element-examples/si-select.spec.ts

```diff
@@ -45,25 +45,46 @@ test.describe('si-select', () => {
   test(example + ' with filter', async ({ page, si }) => {
     await si.visitExample(example);
 
-    await page.locator('.form-check-inline').nth(2).click();
+    await page.getByRole('checkbox', { name: 'With filter' }).check();
+    const formControlSelect = page.getByRole('combobox', { name: 'FormControl' });
```

Comment:
Seems like you are now using a different one. I guess at least here it has to be `Multi select`.

The idea is, that screenshots to no change